mpf/config_spec.yaml

```diff
@@ -2005,6 +2005,7 @@ widgets:
         select_event: single|event_handler|sw_start
         abort_event: single|event_handler|sw_esc
         force_complete_event: single|event_handler|None
+        block_on_flipper_cancel: single|bool|false
```

Comment:
I don't understand this line. How is text_input related to carousel?

Reply:
Nothing :)

I have a similar implementation in MC for the text input to block on cancel, but MC uses the config_spec from MPF so it will require a change to this file. Thought I might as well take care of it in this PR.